tests/test_examples.py

```diff
@@ -310,11 +315,15 @@ def test(self):
                 env_variables["DEEPSPEED_HPU_ZERO3_SYNC_MARK_STEP_REQUIRED"] = "1"
                 env_variables["PT_HPU_MAX_COMPOUND_OP_SYNC"] = "1"
                 env_variables["PT_HPU_MAX_COMPOUND_OP_SIZE"] = "1"
+            elif ("llama" in model_name) and fsdp:
```

Comment:
Can this be made generic so that we can use fsdp with other models in future? You can test with BERT for now, in future we may have more.